homeassistant/components/unifi/config_flow.py

```diff
@@ -65,10 +76,36 @@ def async_get_options_flow(config_entry):
 
     def __init__(self):
         """Initialize the UniFi flow."""
-        self.config = None
+        self.config = {}
         self.desc = None
         self.sites = None
 
+    async def async_step_ssdp(self, discovery_info):
+        """Handle a discovered unifi device."""
+        parsed_url = urlparse(discovery_info[ssdp.ATTR_SSDP_LOCATION])
+        friendly_name = discovery_info[ssdp.ATTR_UPNP_FRIENDLY_NAME]
+        model_description = discovery_info[ssdp.ATTR_UPNP_MODEL_DESCRIPTION]
+        mac_address = format_mac(discovery_info[ssdp.ATTR_UPNP_SERIAL])
+
+        self.config = {
+            CONF_HOST: parsed_url.hostname,
+        }
+
+        if self._host_already_configured(self.config[CONF_HOST]):
```

Comment:
If UniFi cloud key or controller later also can be discovered using SSDP we take the decision to only discover once is the logical thing here

Reply:
The goal here was to prevent discovery for existing entries since they won't have a unique id

Reply:
Ok!